benchmarks/run_benchmark.py

```diff
@@ -112,6 +112,7 @@ def load_config(scenario_file, model_file, updater_file, dataset_file):
             instance_type="ml.g4dn.xlarge",
             n_workers=1,
             max_time=args.max_time,
+            instance_max_time=args.max_time,
```

Comment:
difference to `max_time`?

Reply:
max_time refers to the tuning time per update step. instance_max_time refers to after which time the sagemaker training job is aborted.